dockerfiles/models/requirements.txt

```diff
@@ -1,4 +1,4 @@
 gnureadline~=8.0
-transformers~=3.0
+transformers~=4.2
```

Comment:
We might want to get @guy1992l and @aviaIguazio on board in regards the "models" images and packages which will require tests etc.